src/Transport/obsoletes.cs

```diff
@@ -134,6 +134,15 @@ public AzureServiceBusQueueSettings EnableExpress(Func<string, bool> condition,
         {
             throw new NotImplementedException();
         }
+
+        /// <summary>
+        /// <remarks> Default is 10.</remarks>
+        /// </summary>
+        [ObsoleteEx(Message = ObsoleteMessages.DeprecatedAndNoLongerRequired, TreatAsErrorFromVersion = "8.0", RemoveInVersion = "9.0")]
```

Comment:
Should we add here a specific deprecation message? Like

> MaxDeliveryCount is no longer required. Delivery Count can be controlled by using `Recoverability()...`

or something similar